### DISCUSSION TITLE
Turkish Character Support

### DISCUSSION THREAD

**Discussion by niyazialpay** [meilisearch/product#755] [Feedback & Feature Proposal]
Hello,

I am experiencing issues with content that contains Turkish characters in Meilisearch. As you can see in the screenshot below:

![image](https://github.com/meilisearch/product/assets/8478032/50803d52-94fc-41dc-b200-c9c11aca0e10)

![image](https://github.com/meilisearch/product/assets/8478032/382e1b0b-3d7b-48e0-8e9c-144bb7636e7b)

When I search for the value recorded as "Işıklı" using lowercase letters, I have to type the entire word. When I search for "ışık," no results are found. When I search for "işık," results are found. However, results should be found for both searches. It seems there is an issue with Turkish characters. How can we solve this problem?

![image](https://github.com/meilisearch/product/assets/8478032/847c7dce-0f6c-4a9a-af5a-8701efac9569)

**Comment by ManyTheFish** [comment]
Hello @niyazialpay,
I don't have any workaround for you on the current Meilisearch version, sorry.
However, Meilisearch relies on [Charabia to Tokenize texts](https://github.com/meilisearch/charabia), so if you have an exhaustive list of compatible characters to convert,
I encourage you to:
1) [Create an issue on the repository](https://github.com/meilisearch/charabia/issues) explaining how the Turkish character should be normalized, I will tag it as a `good first issue` for the community
2) If you want to [contribute](https://github.com/meilisearch/charabia/blob/main/CONTRIBUTING.md) to a Rust repository, Charabia is one of the easiest, and I'd be happy to help you if you need 😄

**Reply by niyazialpay** [reply]
I have created issue, thank you

**Reply by ManyTheFish** [reply]
Thank you @niyazialpay!
